crates/swc/src/config/mod.rs

```diff
@@ -1136,6 +1274,8 @@ pub struct BuiltInput<P: swc_ecma_visit::Fold> {
     /// `pass`.
     pub minify: bool,
     pub external_helpers: bool,
+    /// Path to the source map file.
```

Comment:
```suggestion

```

Reply:
I did exactly as you said. Thank you for your explanation.